fedmsg_meta_fedora_infrastructure/fasshim.py

```diff
@@ -90,6 +90,50 @@ def avatar_url_from_email(email, size=64, default='retro', dns=False):
         hash = md5(email.encode('utf-8')).hexdigest()
         return "https://seccdn.libravatar.org/avatar/%s?%s" % (hash, query)
 
+def make_fasjson_cache(**config):
+    global _fas_cache
+    if _fas_cache:
+        return _fas_cache
+    
+    log.warn("No previous fasjson cache found. Looking to rebuild.")
+
+    try:
+        import fasjson_client
+    except ImportError:
+        log.warn("No fasjson-client installed.  Not caching fasjson.")
+    
+    creds = config['fas_credentials']
+    
+    default_url = 'https://admin.fedoraproject.org/accounts/'
```

Comment:
Yeah, I am leaving all the apps with the defaults they have currently so that it will be easier to find. Also, I am changing the deployment config for each one locally anyway.